lexxpluss_apps/src/actuator_controller.cpp

```diff
@@ -814,16 +837,34 @@ int cmd_duty_rep_all(const shell *shell, size_t argc, char **argv)
 
 int cmd_init(const shell *shell, size_t argc, char **argv)
 {
+    if (argc != 1 && argc != 4) {
```

Comment:
```suggestion
    if (argc != 1 && argc != (ACTUATOR_NUM+1)) {
```

ACTUATOR_NUMが変わる可能性や、4の意味合いを示す上で、この方が良いと思います。
I think this is better for showing the possibility of ACTUATOR_NUM changing and the implications of 4.